test/library/standard/BigInteger/bigintops.graph

```diff
@@ -0,0 +1,11 @@
+perfkeys: Elapsed 256 add assign time =, Elapsed 1024 add assign time =, Elapsed 256 pow time =
```

Comment:
Are we planning to plot 64-bit results too?

Reply:
I had thought we had decided only 256 and 1024 should be plotted to keep the size of the graph down (and 64 bit and 256 bit seem to perform pretty similarly for all except `pow`), but I can add 64-bit in if you think that would also be valuable.

Reply:
I thought 64-bit was the most important to plot since it would be the one most impacted by any overhead.

Reply:
OK, added.